tsconfig.json

```diff
@@ -8,7 +8,7 @@
     // "incremental": true,                   /* Enable incremental compilation */
     "target": "ES6",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
     "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
-    // "lib": [],                             /* Specify library files to be included in the compilation. */
+    "lib": ["DOM","ES6","ES2020"],                             /* Specify library files to be included in the compilation. */
```

Comment:
why do we need ES2020?

Reply:
Using `Promises.allSettled` somewhere in the code

https://stackoverflow.com/questions/60276013/property-allsettled-does-not-exist-on-type-promiseconstructor-ts2339